oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -271,6 +341,16 @@ static int mca_spml_ucx_component_fini(void)
     if(!mca_spml_ucx.enabled)
         return OSHMEM_SUCCESS; /* never selected.. return success.. */
 
+    if (mca_spml_ucx.async_progress) {
+        opal_event_evtimer_del(mca_spml_ucx.tick_event);
```

Comment:
maybe opal_event_evtimer_del() should be after opal_progress_thread_finalize()?
what if the thread is still running and we delete the event?